BROWN2GREEN.md

```diff
@@ -0,0 +1,13 @@
+Existing, or brownfield, services are a significant concern when attempting to adopt modern service delivery practices and technologies. 
+
+"How do I move all of this?", Big Bang thinking, is part of the process that needs to be undone. 
```

Comment:
I would suggest to say, how do I move all or any of these?

Reply:
My intent is that too often folks are concerned about moving everything to the new architecture all at once. The preferred approach would be to think about "how can I move a piece at a time?" So moving "any of these" is aligned with the preferred approach.